src/Umbraco.Cms.Integrations.Commerce.Shopify/Client/config.outputPath.js

```diff
@@ -1 +1 @@
-export const outputPath = 'Release' !== 'Release' ? '../wwwroot' : '../obj/Release/net8.0/clientassets'
```

Comment:
Do we need to keep this file or it can be removed?

Reply:
You can ignore this as I'll correct it for _Shopify_ through a different PR.